WcaOnRails/app/assets/stylesheets/competition_tabs.scss

```diff
@@ -35,6 +35,16 @@ table#competition-tabs {
   }
 }
 
+// By default hide the div holding cutoff/time limit information.
+// If the Events tab or the Schedule tab is active, display it.
+div.time-limit-information {
+  display: none;
+  #competition-events.active ~ &,
+  #competition-schedule.active ~ &, {
+    display: block;
+  }
+}
+
```

Comment:
This could also be written like so:
```css
:not(#competition-events.active):not(#competition-schedule.active) ~ div.time-limit-information {
  display: none;
}
```
(It's not the case here, but generally I like to avoid `display: block;` not to accidentally override a `display` rule somewhere else.)

Reply:
Ah, apparently you cannot use multiple selectors in the `:not` pseudo-class (you can pass them comma separated, but that's experimental).

We could assume that there's always an active tab and write it like so:
```css
.tab-pane.active:not(#competition-events):not(#competition-schedule) ~ div.time-limit-information {
  display: none;
}
```
Whatever you prefer ^^

Reply:
Awesome :)
I also think it's better to not have this default "hide but explicitly show for some cases" and rely on a "hide for some cases", I just didn't find a way to express it properly!